kopykat-ksp/src/test/kotlin/at/kopyk/AnnotationTest.kt

```diff
@@ -15,7 +15,7 @@ class AnnotationTest {
       |val p1 = Person("Alex", 1)
       |val p2 = p1.copyMap(age = { it + 1 })
       |val r = p2.age
-      """.evalsWithArgs(mapOf(KopyKatOptions.GENERATE to KopyKatGenerate.ANNOTATED), "r" to 2)
+      """.evalsWithArgs(mapOf("generate" to KopyKatGenerate.ANNOTATED), "r" to 2)
```

Comment:
This way we can make the constant private. I prefer not to expose implementation details because of tests. This is why I'm happy to revert this though :)